opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -179,6 +185,21 @@ def __init__(
         self._shutdown_once = Once()
         self._shutdown = False
 
+        self._views = views
+
+        previous_view_names = set()
+        checked_views = []
+
+        for view in self._views:
+            if view._name is not None and view._name in previous_view_names:
```

Comment:
Question: If I have view with name and selection criteria that possibly matches more than one instrument do we still keep it? How are we enforcing this?
> For the registration mechanism described above, where selection is provided via configuration, the SDK MUST NOT allow Views with a specified name to be declared with instrument selectors that select more than one instrument (e.g. wild card instrument name)

Reply:
Most of these configuration errors you can't catch ahead of time because you don't know the instruments you'll have in advance, and more instruments can be added at any time. Practically the only way you can make a view that will never conflict with another is if you don't specify the `name` field. Even a very specific and innocuous view like this:

```py
View(
    name="bar",
    instrument_name="foo",
)
```

can create a conflict if the user later adds a new instrument called "bar".

Reply:
Agree, I suggested that Diego proposes a wording change to spec.

Reply:
Done so [here](https://github.com/open-telemetry/opentelemetry-specification/pull/2372#discussion_r813392937).